README.md

```diff
@@ -60,6 +60,34 @@ An initial run_list for the node can also be passed to the subcommand. Example u
 
     knife bootstrap windows ssh ec2-50-xx-xx-124.compute-1.amazonaws.com -r 'role[webserver],role[production]' -x Administrator -i ~/.ssh/id_rsa
 
+### knife windows cert generate
+
+Generates a certificate(x509) containing a public / private key pair for WinRM 'SSL' communication.
+The certificate will be generated in three different formats *.pfx, *.der and *.pem.
+The PKCS12(i.e *.pfx) will be added to WinRM Server's Certificate Store.
+The *.der is base64 representaion of *.pfx which may be useful in cloud provider api.
+The *.pem contains public key and can be used at client side.
+This command also displays thumbprint of generated certificate.
+
+    knife windows cert generate --cert-passphrase "strong_passphrase" --domain "cloudapp.net" --output-file "C:\Users\Administrator\server_cert.pfx"
+    # This command will generate certificates at "C:\Users\Administrator\" location with names server_cert.der, server_cert.pfx and server_cert.pem.
+
+### knife windows cert install
+
+This command should be run on Windows Platform. The command will add the specified certificate to its certificate store. The command should include valid PKCS12(i.e *.pfx) certificate file path.
```

Comment:
I think a phrase like `This command only functions on Windows` is a little cleaner (same for the next command). Also, we should say **must** instead of *should* in the rest of the comments.